docs/ossm/ossm2-migration/cluster-wide/README.md

```diff
@@ -0,0 +1,384 @@
+
+##### Create your Istio resource
+1. Find a namespace with 2.6 control plane:
+
+    ```sh
+    oc get smcp -A
+    NAMESPACE      NAME                   READY   STATUS            PROFILES      VERSION   AGE
+    istio-system   install-istio-system   6/6     ComponentsReady   ["default"]   2.6.4     115m
+    ```
+1. Prepare the `Istio` resource yaml named `ossm-3.yaml` to be deployed to the same namespace as the 2.6 control plane:
+
+    Here we are not using any `discoverySelectors` so the control plane will have access to all namespaces. In case you want to define `discoverySelectors`, keep in mind that all data plane namespaces you are planning to migrate from 2.6 must be matched.
+
+    We don't want the new control plane to inject proxies to workloads in namespaces with `istio-injection=enabled` label at this point so we can't use `default` name and we can't create `default` revision tag at this point.
+
+    ```yaml
+   apiVersion: sailoperator.io/v1alpha1
+   kind: Istio
+   metadata:
+     name: ossm-3 # the name, updateStrategy and version are significant for injection labels
+   spec:
+     updateStrategy:
+       type: RevisionBased
+     namespace: istio-system # 3.0 and 2.6 control planes must run in the same namespace
+     version: v1.24.1
+   ```
+1. Apply the `Istio` resource yaml:
+
+    ```sh
+    oc apply -f ossm-3.yaml
+    ```
+1. Verify that new `istiod` is using existing root certificate:
+
+    ```sh
+    oc logs deployments/istiod-ossm-3-v1-24-1 -n istio-system | grep 'Load signing key and cert from existing secret'
+    2024-12-18T08:13:53.788959Z	info	pkica	Load signing key and cert from existing secret istio-system/istio-ca-secret
+    ```
+##### Migrate Workloads
+1. Apply correct labels to the data plane namespace
+
+    Here we're adding two and removing one label:
+
+    1. The `istio.io/rev=ossm-3-v1-24-1` label which ensures that any new pods that get created in that namespace will connect to the 3.0 proxy. In our example, the 3.0 revision is named `ossm-3-v1-24-1`
+    1. The `maistra.io/ignore-namespace: "true"` label which will disable sidecar injection for 2.6 proxies in the namespace. This ensures that 2.6 will stop injecting proxies in this namespace and any new proxies will be injected by the 3.0 control plane.
+    1. It's necessary to temporarily remove the `istio-injection=enabled` as it would prevent the proxy injection by 3.0 control plane.
+
+    ```sh
+    oc label ns bookinfo istio.io/rev=ossm-3-v1-24-1 maistra.io/ignore-namespace="true" istio-injection- --overwrite=true
```

Comment:
Can the `maistra.io/ignore-namespace="true"` label be removed after the migration is complete? Question applies to multi-tenant as well. cc @nrfox

Reply:
It can be removed only after the OSSM 2.6 control planes are removed.

Reply:
We should probably include this to cleaning 2.6 chapter.

Reply:
Done in https://github.com/openshift-service-mesh/sail-operator/pull/202